### ISSUE TITLE
Validator keys import command fails while using testnet-dir

### ISSUE THREAD

**Issue by krogla** [sigp/lighthouse#1992]
## Description

I am trying to import the generated validator keys on my local testnet using the command:
```bash
      docker run -it -v "$PWD/data:/data" sigp/lighthouse lighthouse \
        account validator import --reuse-password --stdin-inputs \
        --datadir "/data/validators" --directory "/data/validator_keys" --testnet-dir "/data/testnet"
```

But I get an error:
```
error: The argument '--testnet-dir <DIR>' cannot be used with '--network <network>'
```

Previously, builds before version 1.0 did not produce such a bug.

## Version

Used latest docker image 
```
Digest: sha256:890656f4fe7dcc4b221f030f086b7df9c0d2d6547f28659c7b3280329ef1413e
```

```
Lighthouse v1.0.0-c6baa0ee+
BLS Library: blst-portable`
```

## Present Behaviour

Validator keys import command fails while specifying `--testnet-dir` option and not specifying `--network`

## Expected Behaviour

Should run without error due to `--network` option is absent.

## Steps to resolve

🤷

**Comment by krogla** [comment]
#1991

**Comment by AgeManning** [comment]
If this ran earlier, it shouldn't have. I don't think it makes sense to run `--testnet-dir`  with `--network`. 

What are you trying to do? 

The `--testnet-dir` should point to a set of configuration files that define the network. The `--network` flag specifies these configuration files with those built into the client. You can/should only have one set of configuration files to define what network you want to run, not both.

**Comment by AgeManning** [comment]
Yep, this seems like the `--network` is being specified regardless when it shouldn't. 

@divagant-martian is looking into this

**Comment by realbigsean** [comment]
fixed in #2029
